packages/x-date-pickers/src/internals/models/validation.ts

```diff
@@ -61,10 +61,12 @@ export interface BaseDateValidationProps<TDate extends PickerValidDate>
   extends FutureAndPastValidationProps {
   /**
    * Maximal selectable date.
+   * @default new Date('2099-12-31T00:00:00.000')
```

Comment:
Do you think that showing a `new Date()` could lead people to pass a `Date` even when using another date library?

Reply:
Yeah, I also fear about it a bit... 🙈 
Do you think only listing `'2099-12-31T00:00:00.000'` or `'2099-12-31'` would be enough? 🤔

Reply:
I don't have a strong opinion, I would go either with:

- `@default 2099-12-31` (no quote to avoid mistakes)
- `@default 2099-12-31 (in date library format, e.g: `dayjs('2099-12-31')` with `AdapterDayjs`)

Reply:
```
@default adapter.date('2099-12-31')
```

This would of course be the most valid option but I don't think it's a good idea

Reply:
I've gone with just `1970-01-01`.
I think it should be clear enough while also avoiding ambiguity. 🤞